app/controllers/TagManagementApi.scala

```diff
@@ -227,12 +231,14 @@ object TagManagementApi extends Controller with PanDomainAuthActions {
     }
   }
 
-  def deleteTag(id: Long) = (APIAuthAction andThen DeleteTagPermissionsCheck).async { req =>
-    implicit val username = Option(req.user.email)
-    (new DeleteTagCommand(id)).process.map{ result =>
-      result.map{t => NoContent } getOrElse NotFound
-    } recover {
-      commandErrorAsResult
+  def deleteTag(id: Long) =  CORSable(conf.corsablePostDomains: _*) {
+    (APIAuthAction andThen DeleteTagPermissionsCheck).async { req =>
```

Comment:
If we're planning to have a background process in Targeting delete tags in tag manager then we might need to make this an `APIHMACAuthAction`